pkgs/misc/anarchism/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name = "anarchism-${version}";
+  version = "15.0";
+  src = fetchurl {
+    url = https://0xacab.org/ju/afaq/repository/master/archive.tar.gz;
```

Comment:
This will break very soon. You should provide a link to a stable archive, i.e. one that will never change. Or use `fetchgit`.

Reply:
done!